nixos/modules/services/web-apps/jitsi-meet.nix

```diff
@@ -265,14 +278,21 @@ in
 
     systemd.services.jitsi-meet-init-secrets = {
       wantedBy = [ "multi-user.target" ];
-      before = [ "jicofo.service" "jitsi-videobridge2.service" ] ++ (optional cfg.prosody.enable "prosody.service");
+      before = [ "jicofo.service" "jitsi-videobridge2.service" ] ++ (optional cfg.jigasi.enable "jigasi.service") ++ (optional cfg.prosody.enable "prosody.service");
       path = [ config.services.prosody.package ];
       serviceConfig = {
         Type = "oneshot";
       };
 
       script = let
-        secrets = [ "jicofo-component-secret" "jicofo-user-secret" "jibri-auth-secret" "jibri-recorder-secret" ] ++ (optional (cfg.videobridge.passwordFile == null) "videobridge-secret");
+        secrets =
+          [ "jicofo-component-secret"
+            "jicofo-user-secret"
+            "jibri-auth-secret"
+            "jibri-recorder-secret"
+            "jigasi-user-secret"
+            "jigasi-component-secret"
+          ] ++ (optional (cfg.videobridge.passwordFile == null) "videobridge-secret");
```

Comment:
When this PR branch is rebased to master there will be a merge conflict in this file. I believe it is primarily because of this recent commit implementing some of the same updates you did for the PR, but in a different way: https://github.com/NixOS/nixpkgs/commit/aea940da63c27f503915a5ca44cbb9eddf2674f6 
As part of the rebase for my transcription branch, this is how I modified your commit to merge the two versions: https://github.com/cleeyv/nixpkgs/commit/7778f69c0bb323e7cae7caf3581da7a633bd8962

Reply:
```suggestion
           ] ++ optional (cfg.videobridge.passwordFile == null) "videobridge-secret"; 
```